### DISCUSSION TITLE
Linkerd Jaeger ImagePullSecrets

### DISCUSSION THREAD

**Discussion by Jaxwood** [linkerd/linkerd2#7977] [Q&A] [answered]
I've deploy the [stable-2.11.1 ](https://github.com/linkerd/linkerd2/tree/stable-2.11.1) Helm chart for [Linkerd2](https://github.com/linkerd/linkerd2/tree/stable-2.11.1/charts/linkerd2) including [Viz](https://github.com/linkerd/linkerd2/tree/stable-2.11.1/viz/charts/linkerd-viz) and [Jaeger](https://github.com/linkerd/linkerd2/tree/stable-2.11.1/jaeger/charts/linkerd-jaeger).

I updated all the images to point to my private registry and this worked fine for Linkerd2 and Viz Charts since they support putting in imagepullsecrets. For Jaeger I don't see the same configuration being possible.

Is it me missing something or is that not currently supported?

In case it is missing - I would like to contribute to the codebase with this functionality.

Thanks

**Comment by alpeb** [comment, marked as answer]
Hi @Jaxwood , you're right, that functionality was missed in the Jaeger extension. We would love indeed if you could contribute with that :+1:

**Reply by Jaxwood** [reply]
@alpeb I've created #7979 - let me know if it is OK
